org.nongnu.gsequencer.gsequencer.json

```diff
@@ -182,8 +182,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/steinbergmedia/vst3sdk.git",
-                    "tag": "v3.7.2_build_28",
-                    "commit": "138ac4eb7332d44873fe8ca2e326887ab8ef671a"
+                    "tag": "v3.7.7_build_19",
+                    "commit": "358b72ee61bc67fb4592b0d492e0c6a1211ebf11"
```

Comment:
the build fail because this needs some extra patches to build on a recent gcc, like `#include <limits>` in the right place.

Reply:
Yes, I just figured out missing limits and it needs root.